src/pivot-table/hooks/__tests__/use-column-width.test.ts

```diff
@@ -352,7 +352,7 @@ describe("useColumnWidth", () => {
       layoutService.layout.qHyperCube.qMeasureInfo = [meaInfo, meaInfoPixels, meaInfoFitToContent];
 
       const { getRightGridColumnWidth } = renderUseColumnWidth();
-      expect(getRightGridColumnWidth(0)).toBe(98);
+      expect(getRightGridColumnWidth(0)).toBe(132);
```

Comment:
Now that the left column width is determined by the bottom row again, resetting this value to what it was before this https://github.com/qlik-oss/sn-pivot-table/pull/434/files#diff-f85cc6b2e82b4ca1f93f5d8293113f571eb9b79d2540e85ce634d378e7ba28e3R355